api/tests/routes/native/v1/account_test.py

```diff
@@ -776,35 +799,15 @@ def test_update_email_errors(self, client, app, email, password):
             },
         )
 
-        assert response.status_code == 400
+        assert response.status_code == response_status
+        if expected_error_code:
+            assert response.json["error_code"] == expected_error_code
+        if expected_error_message:
+            assert response.json["message"] == expected_error_message
 
         user = users_models.User.query.filter_by(email=self.identifier).first()
         assert user.email == self.identifier
-        assert not mails_testing.outbox
-
-    def test_update_email_existing_email(self, app, client):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Pourquoi tu supprimes ce test ?
Le modifier pourquoi pas, mais c'est un vrai cas à tester